pkg/validation/internal/operatorhub.go

```diff
@@ -215,40 +215,9 @@ func validateBundleOperatorHub(bundle *manifests.Bundle, k8sVersion string) erro
 		result.Add(errors.WarnFailedValidation(warn.Error(), bundle.CSV.GetName()))
 	}
 
-	if warn := validateHubChannels(bundle.Channels); warn != nil {
-		result.Add(errors.WarnFailedValidation(warn.Error(), bundle.CSV.GetName()))
-	}
-
 	return result
 }
 
-// validateHubChannels will check the channels. The motivation for the following check is to ensure that operators
-// authors knows if their operator bundles are or not respecting the Naming Convention Rules.
-// However, the operator authors still able to choose the names as please them.
-func validateHubChannels(channels []string) error {
```

Comment:
Just to be clear, we're moving this to the "good practices" file since these are suggested best channel naming conventions, right?

Reply:
Yes. It was added to the OperatorHub because we had not the good practices one.
However, it shows fit better in this Validator since we might want to call only this one to check all recommendations related to common suggestions and good practices.